packages/components/src/components/dialog/dialog.jsx

```diff
@@ -32,6 +32,8 @@ const Dialog = ({
         portal_element_id,
         title,
         has_close_icon,
+        secondary_button_type,
+        primary_button_type,
```

Comment:
Why do we need these props? we already have `primary` and `secondary` props on the buttons.

Reply:
Yes, we have such props for buttons, and we have `type` prop for button also, but we can't change these props inside `Dialog` component. And without pointing the type of buttons anytime while clicking buttons in Dialog window they will work like submit buttons and call the validation on token form. But now after passing `portal_element_id` to the dialog and using it with portal I don’t see such behaviour. So I'll delete those addition props. :+1: